template/src/routes/notfound/style.css

```diff
@@ -0,0 +1 @@
+.notfound{}
```

Comment:
i thought this had some css in the other PR

Reply:
done